public/pages/CreateTrigger/containers/CreateTrigger/utils/triggerToFormik.js

```diff
@@ -81,3 +164,51 @@ export function triggerToFormik(trigger, monitor) {
     },
   };
 }
+
+export function getAggregationTriggerConditions(condition) {
+  const triggerConditions = segmentArray(condition.script.source, 4).map((conditionArray) =>
+    convertToTriggerCondition(conditionArray, condition)
+  );
+
+  return triggerConditions;
+}
+
+export function convertToTriggerCondition(conditionArray, condition) {
+  const { buckets_path, gap_policy, parent_bucket_path, script } = condition;
+  const enumOptions = {
+    '>': 'ABOVE',
+    '<': 'BELOW',
+    '==': 'EXACTLY',
+  };
+
+  const queryMetric = conditionArray[0];
```

Comment:
Also there is a `params.` here we want to remove that was put there when it became a Trigger condition. If we don't it'll keep appending `params.` to it when it gets edited and saved.